libs/application/templates/general-fishing-license/src/lib/messages/fishingLicense.ts

```diff
@@ -100,6 +100,11 @@ export const fishingLicense = {
       defaultMessage: 'Krabbaveiðileyfi',
       description: 'Crustaceans',
     },
+    igulker: {
+      id: 'gfl.application:shipSelection.labels.igulker',
+      defaultMessage: 'Ígulkeraveiðileyfi',
+      description: 'Ígulkeraveiðileyfi',
```

Comment:
```suggestion
      description: 'Ígulkerjaveiðileyfi',
```
Declines with "j" with genitive case
https://is.wiktionary.org/wiki/%C3%ADgulker